script.js

```diff
@@ -0,0 +1,91 @@
+const body = document.querySelector('body');
+
+const textArea = document.createElement('textarea');
+textArea.className = 'board';
+body.append(textArea);
+
+const language = document.createElement('div');
+language.className = 'language';
+language.innerHTML = 'Привет!) Чтобы поменять раскладку клавиатуры - нажмите Томатную кнопку и поменяйте раскладку на вашем устройстве)';
+body.append(language);
+
+const keyboard = document.createElement('div');
+keyboard.className = 'keyboard';
+keyboard.id = 'keyboard';
+body.append(keyboard);
+
+const keyboardNumbersRus = [62, 49, 50, 51, 52, 53, 54, 55, 56, 57, 48, 45, 61, 1081,
```

Comment:
Magic Numbers

Reply:
It's charCode of buttons. I've just corrected the name of this variable